dj/models/engine.py

```diff
@@ -1,14 +1,26 @@
 """
 Models for columns.
 """
-
+import enum
 from typing import Optional
 
+from sqlalchemy.sql.schema import Column as SqlaColumn
+from sqlalchemy.types import Enum
 from sqlmodel import Field, SQLModel
 
 from dj.models.base import BaseSQLModel
 
 
+class Dialect(str, enum.Enum):
+    """
+    SQL dialect
+    """
+
+    SPARK = "spark"
+    TRINO = "trino"
+    POSTGRES = "postgres"
```

Comment:
Does this translate to the lowercase or uppercase string when saved to the backend? I noticed the enum defined in the alembic migration uses uppercase so I'm wondering if these need to be `TRINO = "TRINO"` for example.

Reply:
So I think what happens here is that the database engine translates everything to lowercase (we have the same pattern for NodeType, where the alembic migration uses uppercase but the enum has lowercase). Although perhaps we should just be consistent in the migration and have these be lowercase to make it clearer. Let me regenerate